conbencher/python/setup.py

```diff
@@ -0,0 +1,38 @@
+import pathlib
+
+import setuptools
+
+repo_root = pathlib.Path(__file__).parent
+
+with open(repo_root / "README.md", "r") as readme:
+    long_description = readme.read()
+
+__version__ = ""
+with open(repo_root / "conbencher" / "_version.py", "r") as f:
+    exec(f.read())  # only overwrites the __version__ variable
+
+install_requires = [
+    line.strip()
+    for line in pathlib.Path(__file__)
+    .parent.joinpath("requirements.txt")
+    .read_text()
+    .splitlines()
+]
+
+setuptools.setup(
+    name="conbencher",
+    version=__version__,
+    description="Utilities for Conbench Runs",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3.8",
+        "License :: OSI Approved :: MIT License",
```

Comment:
I think we want this to be apache2 to match all of the other modules (I know conbench itself is something else, but all new things should be apache2 ideally)

Reply:
Ah, I definitely copied this from somewhere...maybe benchmarks or conbench 😬 Will switch

Reply:
Changed this and added a license file. Do we need a `NOTICE` file? Apache suggests yes, but nobody else seems to have one.